server/src/main/java/org/elasticsearch/action/admin/indices/rollover/MetadataRolloverService.java

```diff
@@ -84,6 +84,11 @@ private RolloverResult(String rolloverIndexName, String sourceIndexName, Cluster
             this.sourceIndexName = sourceIndexName;
             this.clusterState = clusterState;
         }
+
+        @Override
+        public String toString() {
+            return String.format(Locale.ROOT, "rollover index name [%s], source index name [%s]", rolloverIndexName, sourceIndexName);
```

Comment:
Can you add the cluster stat version to this, perhaps:

```suggestion
            return String.format(Locale.ROOT, "[%d] rollover index name [%s], source index name [%s]", clusterState.version(), rolloverIndexName, sourceIndexName);
```

(or some other format)